### PR TITLE
build: Also search PATH for rcedit 

### PR DESCRIPTION
**Pull request by kwkam**
## PR Summary

Allow custom location for `rcedit.exe`

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: build.psm1 [hunk 1]
```diff
@@ -602,7 +602,11 @@ Fix steps:
 
     if ($Environment.IsWindows) {
         ## need RCEdit to modify the binaries embedded resources
-        if (-not (Test-Path "~/.rcedit/rcedit-x64.exe")) {
+        $rcedit = "~/.rcedit/rcedit-x64.exe"
+        if (-not (Test-Path -Type Leaf $rcedit)) {
+            $rcedit = Get-Command rcedit-x64 -ErrorAction SilentlyContinue | Select-Object -First 1 | % Name
```
Comment (iSazonov) [inline build.psm1]: Should it be `Get-Command rcedit-x64.exe -CommandType Application -ErrorAction SilentlyContinue `?

Comment (kwkam) [reply build.psm1]: Yes, that's better.

**Review by iSazonov — APPROVED** [review]

**Review by dantraMSFT — APPROVED** [review]
Minor note: I suggest changing the PR description to something indicating the PATH is used as a fallback. Something like 'Use PATH to find rdedit if not found in the expected location.'

Other than that, LGTM.

**Comment by TravisEz13** [comment]
VSTS issue is known and fixed.  ignoring since we are still running in parallel with travis-ci

**Pull request merged**